### PR TITLE
DCA-137: Refactor the creation reports code and logic

### PR DESCRIPTION
**Pull request by astashys**
https://bulldog.internal.atlassian.com/browse/DCA-137

### CODE REVIEW

File: jira/util/reports_generation/README.md [hunk 1]
```diff
@@ -1,16 +1,36 @@
-## CSV report aggregator and chart generation tool
+## Reports generator - tool that creates raw data (aggregates results.csv files) and charts
 Before you start, make sure you have installed Python packages from `jira/requirements.txt`.
 
 Otherwise, run the `pip install -r requirements.txt` command from `jira` directory to install necessary packages to your virtual environment.
 
-To create aggregated csv report and chart with one command (recommended), run the <br> 
-`python csv_chart_generator.py [performance_profile.yml or scale_profile.yml]` command from the `jira/util/reports_generation` folder.
+To create reports, run the <br>
+`python reports_generator.py [performance_profile.yml or scale_profile.yml]` command from the `jira/util/reports_generation` folder.
```
[the remark this answers is not in the document]
Comment (astashys) [reply jira/util/reports_generation/README.md]: reworked

File: jira/util/reports_generation/README.md [hunk 2]
```diff
@@ -1,16 +1,36 @@
...
 
-To create aggregated csv report or chart separately:
-* To create only an aggregated csv report, run the <br>
-`python csv_aggregator.py [perfomance_profile.yml or scale_profile.yml]` command from the `jira/util/reports_generation/csv_aggregator` folder.
+The raw data and chart are stored in the `jira/results/reports` directory.
```
[the remark this answers is not in the document]
Comment (astashys) [reply jira/util/reports_generation/README.md]: reworked

File: jira/util/reports_generation/README.md [hunk 3]
```diff
@@ -1,16 +1,36 @@
...
 
-* To create only a chart based on already created aggregated csv report, run the <br>
-`python csv_chart_generator.py [perfomance_profile.yml or scale_profile.yml]` command from the `jira/util/reports_generation/chart_generator` folder.
+#####Configuration
```
[the remark this answers is not in the document]
Comment (astashys) [reply jira/util/reports_generation/README.md]: removed yml config example from readme

File: jira/util/reports_generation/performance_profile.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Defines which column from test runs is used for aggregated report. Default is "90% Line"
 column_name: "90% Line"
 runs:
-   # fullPath should contain full path to directory with results.csv. E.g. /home/$USER/centaurus/jira/results/2019-08-06_17-41-08
+   # fullPath should contain full path to results.csv. E.g. /home/$USER/centaurus/jira/results/2019-08-06_17-41-08/results.csv
```
[the remark this answers is not in the document]
Comment (astashys) [reply jira/util/reports_generation/performance_profile.yml]: reworked

File: jira/util/reports_generation/README.md [hunk 5]
```diff
@@ -1,16 +1,19 @@
...
-`python csv_aggregator.py [perfomance_profile.yml or scale_profile.yml]` command from the `jira/util/reports_generation/csv_aggregator` folder.
+The aggregated .csv files and charts are stored in the `jira/results/reports` directory.
+Before run, you should edit `performance_profile.yml` or `scale_profile.yml` and set appropriate `fullPath` values. 
 
-* To create only a chart based on already created aggregated csv report, run the <br>
-`python csv_chart_generator.py [perfomance_profile.yml or scale_profile.yml]` command from the `jira/util/reports_generation/chart_generator` folder.
-
-The aggregated csv report and chart are stored in the jira/util/reports_generation/results directory.
+**Configuration**
+- `column_name` - means column from `results.csv` to comparison.
```
Comment (ometelytsia) [inline jira/util/reports_generation/README.md]: - `column_name` - column name from `results.csv` used for aggregation
- `run_name` - label for specific run
- `fullPath` -  the full path to result folder of specific run
- `index_col` - index column
- `title` - chart title (also this value is used as file name)
- `image_height_px` - chart image height in pixels
- `image_width_px` - chart image width in pixels

File: jira/util/reports_generation/performance_profile.yml [hunk 6]
```diff
@@ -1,7 +1,7 @@
 # Defines which column from test runs is used for aggregated report. Default is "90% Line"
 column_name: "90% Line"
 runs:
-   # fullPath should contain full path to directory with results.csv. E.g. /home/$USER/centaurus/jira/results/2019-08-06_17-41-08
+   # fullPath should contain full path to results.csv. E.g. /home/$USER/centaurus/jira/results/2019-08-06_17-41-08
```
Comment (ometelytsia) [inline jira/util/reports_generation/performance_profile.yml]: # fullPath should contain a full path to the directory with run resutls. E.g. /home/$USER/dc-app-performance-toolkit/jira/results/2019-08-06_17-41-08

File: jira/util/reports_generation/scale_profile.yml [hunk 7]
```diff
@@ -1,7 +1,7 @@
 # Defines which column from test runs is used for aggregated report. Default is "90% Line"
 column_name: "90% Line"
 runs:
-  # fullPath should contain full path to directory with results.csv. E.g. /home/$USER/centaurus/jira/results/2019-08-06_17-41-08
+  # fullPath should contain full path to results.csv. E.g. /home/$USER/centaurus/jira/results/2019-08-06_17-41-08
```
Comment (ometelytsia) [inline jira/util/reports_generation/scale_profile.yml]: fullPath should contain a full path to the directory with run resutls. E.g. /home/$USER/dc-app-performance-toolkit/jira/results/2019-08-06_18-41-08

File: jira/util/reports_generation/README.md [hunk 8]
```diff
@@ -1,16 +1,36 @@
-## CSV report aggregator and chart generation tool
+## Reports generator - tool that creates raw data (aggregates results.csv files) and charts
```
[the remark this answers is not in the document]
Comment (astashys) [reply jira/util/reports_generation/README.md]: done

File: jira/util/reports_generation/README.md (hunk 5 above)
Comment (astashys) [reply jira/util/reports_generation/README.md]: done

File: jira/util/reports_generation/performance_profile.yml (hunk 6 above)
Comment (astashys) [reply jira/util/reports_generation/performance_profile.yml]: done

File: jira/util/reports_generation/scale_profile.yml (hunk 7 above)
Comment (astashys) [reply jira/util/reports_generation/scale_profile.yml]: done

**Pull request merged**